packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -116,6 +121,29 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
     return () => unregisterEnvAdaptor()
   }, [])
 
+  useEffect(() => {
+    // reconcile local storage state with URL or vice versa
+    if (initialized) {
+      const sdkParam = searchParams.get('sdk') || ''
+      const sdk = findSdk(sdkParam)
+      const paramMatchesSdk =
+        !sdkParam.localeCompare(sdk.alias, 'en', { sensitivity: 'base' }) ||
+        !sdkParam.localeCompare(sdk.language, 'en', { sensitivity: 'base' })
+      if (sdkParam && sdk && paramMatchesSdk) {
```

Comment:
is `sdk` redundant in this check? Also perhaps `sdkParam && validSdkParam` would be more descriptive here.

Reply:
Yeah, I think we can get by with just `validSdkParam` for the conditional check, as it will encapsulate all situations. If there is no sdk parameter, then sdkParam will be '' and the comparison will be false. If there is a parameter then we need to know if it matches what was found when we searched.